notes/stories/film/film-scifi-2010s.st.txt

```diff
@@ -2555,6 +2555,59 @@ intelligent ape
 sign language
 
 
+movie: Starship Troopers: Traitor of Mars (2017)
+================================================
+
+:: Title
+Starship Troopers: Traitor of Mars
+
+:: Description
+It is up to Johnny Rico and his team of young recruits to stop bug aliens
+from overrunning a Federation colony on Mars. It is the fifth installment of
+the Starship Troopers film series.
+
+:: Date
+2017-08-21
+
+:: References
+https://en.wikipedia.org/wiki/Starship_Troopers:_Traitor_of_Mars
+
+:: Ratings
+1 <paul>
+
+:: Genre
+sci-fi
+action
+
+:: Major Themes
+insectoid [Federation forces were battling bug aliens, known as Arachnids.]
+extraterrestrial being [Federation forces were battling bug aliens, known as Arachnids.]
+interstellar war [The story is set in a future where humans are mired in a multiple star systems spanning war with a race of hostile bug aliens, known as the Arachnids.]
+the military professions [One main storyline follows Colonel Johnny Rico as he and his squad of new recruits battle bug aliens on the Federation Mars colony.]
+colonization of Mars [Mars, which in this story is the main theater of war between the Federation and the bug aliens, had been terraformed and colonized by humans.]
+independence struggle [The Mars colonists were engaged in a political movement to be secede from the Federation.]
+the good of the many vs. the needs of the few [The viewer is made to ponder whether the Federation should blow Mars colony to smithereens or stand idly by and allow to be overrun by bug aliens and used by them as a stepping stone to conquering Earth.]
+
+:: Minor Themes
+terraforming [Mars had been terraformed to have a breathable atmosphere.]
+space station [The Mars Orbital Base was a large space station in orbit around Mars called.]
+speculative spaceship [The viewer is shown numerous various Federation battle cruisers over the course of the film. Captain Carmen Ibanez was shown on the bridge of the starship John A. Warden.]
+colonization of the Moon [There was a thriving spaceport on the Moon.]
```

Comment:
Sounds like colonisation of Mars was also on the agenda..?

Reply:
Darn straight, **colonization of Mars** is up there among the major themes.